bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/config/config.xml

```diff
@@ -194,33 +191,33 @@
 
 	<config-description uri="thing-type:shelly:rgbw2">
 		<parameter name="deviceIp" type="text" required="true">
-			<label>Device IP Address</label>
-			<description>IP-Address of the Shelly device.</description>
+			<label>@text/thing-type.config.shelly.deviceIp.label</label>
+			<description>@text/thing-type.config.shelly.deviceIp.description</description>
 			<context>network-address</context>
 		</parameter>
 		<parameter name="userId" type="text" required="false">
-			<label>User</label>
-			<description>User ID for HTTP API access.</description>
+			<label>@text/thing-type.config.shelly.userId.label</label>
+			<description>@text/thing-type.config.shelly.userId.description</description>
 		</parameter>
 		<parameter name="password" type="text" required="false">
-			<label>Password</label>
-			<description>Password for HTTP API access.</description>
+			<label>@text/thing-type.config.shelly.password.label</label>
+			<description>@text/thing-type.config.shelly.password.description</description>
 			<context>password</context>
 		</parameter>
 		<parameter name="brightnessAutoOn" type="boolean" required="false">
-			<label>Brightness Auto-ON</label>
-			<description>true: Turn device ON if brightness > 0 is set; false: don't touch power status when brightness is set.</description>
+			<label>@text/thing-type.config.shelly.light.brightnessAutoOn.label</label>
+			<description>@text/thing-type.config.shelly.light.brightnessAutoOn.description</description>
 			<default>true</default>
 		</parameter>
 		<parameter name="eventsCoIoT" type="boolean" required="false">
-			<label>Enable CoIoT Events</label>
-			<description>If enabled, the CoAP protocol is used to receive status updates.</description>
-			<advanced>true</advanced>
+			<label>@text/thing-type.config.shelly.eventsCoIoT.label</label>
+			<description>@text/thing-type.config.shelly.eventsCoIoT.description</description>
 			<default>true</default>
+			<advanced>true</advanced>
```

Comment:
Maybe the advanced property should be the same everywhere for this parameter ?
I see it sometimes true, sometimes false.